test/fluree/db/query/history_test.clj

```diff
@@ -0,0 +1,88 @@
+(ns fluree.db.query.history-test
+  (:require [clojure.test :refer :all]
+            [fluree.db.test-utils :as test-utils]
+            [fluree.db.json-ld.api :as fluree]))
+
+(deftest ^:integration history-query
+  (let [conn (test-utils/create-conn)
+        ledger @(fluree/create conn "historytest" {:context {:ex "http://example.org/ns/"}})
+
+        db1 @(test-utils/transact ledger {:id :ex/dan
+                                          :ex/x "foo-1"
+                                          :ex/y "bar-1"})
+        db2 @(test-utils/transact db1 {:id :ex/dan
+                                       :ex/x "foo-2"
+                                       :ex/y "bar-2"})
+        db3 @(test-utils/transact db2 {:id :ex/dan
+                                       :ex/x "foo-3"
+                                       :ex/y "bar-3"})
+        db4 @(test-utils/transact db3 {:id :ex/cat
+                                       :ex/x "foo-cat"
+                                       :ex/y "bar-cat"})
+        db5 @(test-utils/transact db4 {:id :ex/dan
+                                       :ex/x "foo-cat"
+                                       :ex/y "bar-cat"})]
+    (testing "subject history"
+      (is (= [{:t 5 :assert {:ex/x "foo-cat" :ex/y "bar-cat"} :retract {:ex/x "foo-3" :ex/y "bar-3"}}
```

Comment:
@jdorety Do you know what sort order is the most useful for history queries? Currently returning most recent first, but if there's a huge convenience in having it go the other way let me know.